crates/sui-framework/tests/math_tests.move

```diff
@@ -18,6 +18,28 @@ module sui::math_tests {
         assert!(math::min(100, 10) == 10, 2);
         assert!(math::min(0, 0) == 0, 3);
     }
+    
+    #[test]
+    fun test_pow() {
+        assert!(nft::pow(2, 10) == 1024, 0);
```

Comment:
Can you also add some test cases for raising to `0`, `1`, mixes of even and odd numbers (to exercise the various paths in the logarithmic algorithm).

Reply:
Well... ignore me.  Looks like I was reviewing an old version of your PR!